src/main/java/org/broadinstitute/hellbender/engine/LocalReadShard.java

```diff
@@ -84,6 +95,13 @@ public void setDownsampler(final ReadsDownsampler downsampler) {
         this.downsampler = downsampler;
     }
 
+    /**
+     * Reads in this shard will be transformed after filtering and downsampling
```

Comment:
Description is no longer accurate: should be "after filtering but before downsampling"

Reply:
done